components/Modal/MemoryDetail.tsx

```diff
@@ -119,23 +128,26 @@ const RemoveMemoryButton = styled(Button)``
 
 const ImageWrapper = styled.div`
   width: 100%;
-  height: 100%;
+  max-width: 500px;
+  height: 435px;
   border-radius: 5px;
-  min-height: 435px;
   display: flex;
   justify-content: center;
   align-items: center;
   background-color: rgba(0, 0, 0, 0.1);
+  overflow: hidden;
+  position: relative;
+  object-fit: contain;
 `
-const MemoryImage = styled.div<{ src: string }>`
-  background-image: url(${(props) => props.src});
-  width: 100%;
-  min-width: 307px;
-  height: 435px;
-  background-size: contain;
-  background-position: center center;
-  background-repeat: no-repeat;
-`
+// const MemoryImage = styled.div<{ src: string }>`
+//   background-image: url(${(props) => props.src});
+//   width: 100%;
+//   min-width: 307px;
+//   height: 435px;
+//   background-size: contain;
+//   background-position: center center;
+//   background-repeat: no-repeat;
+// `
 
 const Date = styled.span`
```

Comment:
요거요거 javascript의 Date 객체랑 이름이 겹칠 수 있어서 안좋은 이름이라고 생각합니당
`DateSpan`같은걸로 변경 어떨까요

```suggestion
const DateSpan = styled.span`
```

Reply:
감사합니다! 수정했어요!